packages/core/apps/core-apps-server-internal/assets/legacy_styles.css

```diff
@@ -0,0 +1,116 @@
+*, *:before, *:after {
+  box-sizing: border-box;
+}
+
+html, body, div, span, svg {
+  margin: 0;
```

Comment:
I extracted the static inline styles (from `packages/core/rendering/core-rendering-server-internal/src/views/styles.tsx`), and added them in a css file instead of having them inlined in the template. Better isolation of concern, and now it's cache-able.